### PR TITLE
Notebooks converter

### PR DESCRIPTION
**Pull request by viciopoli01**
Convert the notebooks into python scritps and copy them into a package in the ros workspace according to the config.yaml

### CODE REVIEW

**Comment by CourchesneA** [comment]
This breaks `dts`:

ERROR:dts:


            !   Could not load commands.

                Cannot load command class 'exercises.test.command.DTCommand' (package='exercises.', command='test'): Traceback (most recent call last):
  File "/home/anthony/.local/lib/python3.8/site-packages/dt_shell/cli.py", line 228, in _load_commands
    klass = _load_class(spec)
  File "/home/anthony/.local/lib/python3.8/site-packages/dt_shell/cli.py", line 336, in _load_class
    mod = __import__(components[0])
  File "/home/anthony/.dt-shell/commands-multi/daffy/exercises/__init__.py", line 47, in <module>
    exec("from .%s import *" % _basename(_mod))
  File "<string>", line 1, in <module>
  File "/home/anthony/.dt-shell/commands-multi/daffy/exercises/test/__init__.py", line 2, in <module>
    from .command import DTCommand
  File "/home/anthony/.dt-shell/commands-multi/daffy/exercises/test/command.py", line 509
    dtslogger.info(f"Containedef convertNotebook(filepath, export_path) -> bool:
                                                                               ^
SyntaxError: EOL while scanning string literal

File: exercises/test/command.py [hunk 1]
```diff
@@ -503,7 +506,22 @@ def launch_container_monitor(containers_to_monitor, stop_attached_container):
     """
     monitor_thread = threading.Thread(target=monitor_containers, args=(containers_to_monitor, stop_attached_container), daemon=True)
     dtslogger.info("Starting monitor thread")
-    dtslogger.info(f"Containers to monitor: {[container.name for container in containers_to_monitor]}")
+    dtslogger.info(f"Containedef convertNotebook(filepath, export_path) -> bool:
```
Comment (CourchesneA) [inline exercises/test/command.py:509]: Did you forget to close the quotation ? Also, not sure you need a f-string here.

Comment (viciopoli01) [reply exercises/test/command.py:509]: I am going to fix that. I don't know why but probably I did `CRTL+V` (?) in a completely wrong place....

File: exercises/test/command.py [hunk 2]
```diff
@@ -503,7 +506,22 @@ def launch_container_monitor(containers_to_monitor, stop_attached_container):
...
+    # source is a tuple of python source code
+    # meta contains metadata
+    source, _ = exporter.from_notebook_node(nb)
+    try:
+        with open(export_path, "w+") as fh:
+            fh.writelines(source)
+    except Exception:
+        return False
+
+    return Truers to monitor: {[container.name for container in containers_to_monitor]}")
```
Comment (CourchesneA) [inline exercises/test/command.py:524]: Also not sure what this is supposed to do, I don't think this should have been merged

Comment (viciopoli01) [reply exercises/test/command.py:524]: Yes that the problem this should be :

```
def launch_container_monitor(containers_to_monitor, stop_attached_container):
    """
    Start a daemon thread that will exit when the application exits.
    Monitor should Stop everything if a containers exits and display logs
    """
    monitor_thread = threading.Thread(target=monitor_containers, args=(containers_to_monitor, stop_attached_container), daemon=True)
    dtslogger.info("Starting monitor thread")
    dtslogger.info(f"Containers to monitor: {[container.name for container in containers_to_monitor]}")
    monitor_thread.start()

```

But accidentally I did something wrong

**Pull request merged**